VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1401,9 +1418,42 @@ Function Install-EsxiCertificate {
                 continue
             }
 
+            if ($vcfVersion -eq "5.2") {
+                $keyPath = Join-Path -Path $certificateDirectory -childPath ($esxiFqdn + ".key")
+                if (!(Test-Path $keyPath -PathType Leaf) -and ($PSBoundParameters.ContainsKey("uploadPrivateKey"))) {
+                    Write-Error "Private key not found at $keyPath. Skipping certificate replacement for ESXi host $esxiFqdn."
+                    $skippedHosts.Add($esxiFqdn)
+                    continue
+                }
+            }
+
             if (Confirm-EsxiCertificateInstalled -server $server -user $user -pass $pass -esxiFqdn $esxiFqdn -signedCertificate $crtPath) {
                 $skippedHosts.Add($esxiFqdn)
                 continue
+            } elseif ($vcfVersion -eq "5.2") {
```

Comment:
-ge ?

Reply:
addressed in commit e6a7844965249faebca841b4da521bef3b7b40ba